src/javafundamentals/lambdas/Ex02LambdaWithFunctionalInterface.java

```diff
@@ -8,4 +8,19 @@
  */
 
 public class Ex02LambdaWithFunctionalInterface {
+    public static void main(String[] args) {
+        Printer printer = new Printer();
+
+        printer.printText("Hello", input -> System.out.println(input));
+    }
+}
+
```

Comment:
When creating a functional interface you can annotate it with 
@FunctionalInterface

Reply:
Okay, thanks